packages/rrweb/src/record/observer.ts

```diff
@@ -546,23 +547,23 @@ function initStyleSheetObserver(
   };
 
   const supportedNestedCSSRuleTypes: {
-    [key: string]: GroupingCSSRuleTypes;
+    [key: string]: win.GroupingCSSRuleTypes;
```

Comment:
This is a type, no need to prefix that
```suggestion
    [key: string]: GroupingCSSRuleTypes;
```